ui/lib/pki/addon/components/pki-key-usage.hbs

```diff
@@ -0,0 +1,75 @@
+{{#let (camelize (concat "show" @group)) as |prop|}}
```

Comment:
New component that handles all the logic of adding the key_usage and ext_key_usage checkbox items to their respective model params. It also handles the CSS grid layout for this group.

Reply:
Adopted this pattern after trying to manage this in the model and the serializer and then chatting to Jordan about this approach. This seems to make more sense because the checkboxes are not really params on the model and so should never show there.